wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/sample/Robot.java

```diff
@@ -91,24 +91,31 @@ public void robotInit() {
 	 */
 	@Override
 	public void autonomous() {
-		String autoSelected = m_chooser.getSelected();
-		// String autoSelected = SmartDashboard.getString("Auto Selector",
-		// defaultAuto);
+		String autoSelected = SmartDashboard.getString("Auto Selector",
+				kDefaultAuto);
 		System.out.println("Auto selected: " + autoSelected);
 
+		// Warning: while disabling motor safety is useful for simple examples,
```

Comment:
If it's not good for production code, should it be in the examples? I think there are plenty of rookie/inexperienced teams that want this sample code to get their robot running, and will use these examples. 

Thoughts?

Reply:
I'd love to rip out SampleRobot completely, to be honest. It encourages bad practices like the one above. We already have a patch planned for beta 4 to deprecate it. Everyone should be using TimedRobot.

Reply:
But it makes stuff like this very easy. Other things a little harder, but a while loop + delay gives you iterative/timedRobot. I would say that sample 'requires' different practices, including disabling motor safety. I wouldn't discourage it here because it is kinda necessary unless you want to put the loops back in (but at that point just use iterative). If this was an iterative example I would say it should be discouraged.

Reply:
Disabling motor safety in general is a bad idea. If that's standard practice in SampleRobot, then we should kill SampleRobot.

Reply:
Let's do it *grabs pitchfork*

Reply:
See #472.

Reply:
Motor Safety adds safety when motors are expected to be updated often and something (eg a blocking method) could interfere. In Sample robot autonomous, neither of those are true. The DriverStation watchdog is still active.

Reply:
I know what it does. It just seems weird to say "hey here's some sample code, but don't use it"

Reply:
My point is the warning is too ominous for SampleRobot (but is appropriate for IterativeRobot).

Reply:
I would say something like "MotorSafety improves safety when motors are updated in loops but is disabled because motor calls are not looped in this autonomous."